node/src/cli.rs

```diff
@@ -120,8 +121,18 @@ pub struct Cli {
     #[clap(flatten)]
     pub run: cumulus_client_cli::RunCmd,
 
-    /// Relaychain arguments
-    #[clap(raw = true)]
+    /// Disable automatic hardware benchmarks.
+    ///
+    /// By default these benchmarks are automatically ran at startup and measure
+    /// the CPU speed, the memory bandwidth and the disk speed.
+    ///
+    /// The results are then printed out in the logs, and also sent as part of
+    /// telemetry, if telemetry is enabled.
+    #[clap(long)]
+    pub no_hardware_benchmarks: bool,
```

Comment:
We might need to instruct community collators to run with this flag from now on, so they don't waste compute on rebenchmarking their machines on every node restart
FYI @grenade

Reply:
It will be showed while start the node. And telemetry server will gather these information.
You can see summary here: https://telemetry.polkadot.io/#stats/0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3

Reply:
Actually there's a subcommand introduced in this release:
```
manta benchmark machine --chain ...
```
But it's being hided, not sure when it will be visible.

Reply:
My point was if we should disable it or tell people to disable. But i think the data will be valuable for Rob to whitelist candidates